cli/bin/cmdtree_dp.rs

```diff
@@ -268,11 +268,24 @@ fn cmd_local() -> Node {
     root
 }
 
-#[allow(unused)]
+fn cmd_frrmi_apply_last() -> Node {
+    let mut root = Node::new("apply");
+    root += Node::new("last-config")
+        .desc("Apply the last config in FRR")
+        .action(CliAction::FrrmiApplyLastConfig as u16);
```

Comment:
Is there a command to display this config before applying it?

Reply:
Yes, you can do `show frrmi last-config`.
The apply command is only meant for troubleshooting.

Reply:
But it will have already been applied...